go/felix/ipsets/utils_for_test.go

```diff
@@ -121,6 +126,9 @@ func (d *restoreCmd) CombinedOutput() ([]byte, error) {
 		d.Dataplane.FailNextRestore = false
 		return nil, errors.New("Simulated failure")
 	}
+	if d.Dataplane.FailAllRestores {
+		return nil, errors.New("Simulated failure")
```

Comment:
Might want to differentiate this error string - e.g. "Simulated failure (FailAllRestores)" - so that you can check that things are failing for the expected reason.